core-services/sync-rover-groups/_config.yaml

```diff
@@ -77,7 +77,6 @@ groups:
     - app.ci
   openstack-k8s-operators-ci:
     clusters:
-    - app.ci
```

Comment:
You still need the group on app.ci if you want to access the images on app.ci.

Reply:
ack